### PR TITLE
Account for windows sleep inaccuracy when testing the stress timer on Windows

### PR DESCRIPTION
**Pull request by jsternberg**

### CODE REVIEW

**Comment by BrannonKing** [comment]
A few thoughts on this. First, newer versions of Windows are more accurate on timings (the comment said Windows XP). Second, newer versions of Windows in conjunction with modern graphics cards and games usually adjust the system accuracy from the default 15ms down to 1ms. Having said that, nobody on windows uses Sleep or clock calls for timing. They all use the CPU frequency counters. This boils down to using the native API call: QueryPerformanceCounter.

**Comment by jsternberg** [comment]
@BrannonKing do you know anything about the proper way to do this? I just found that `time.Sleep()` wasn't working correctly for a unit test and it would occasionally cause a failure on our new Windows CI boxes and this was the solution I found from the Go source code. If there's a more optimal solution we can use, I would be open to doing that instead.

File: stress/stress_test.go [hunk 1]
```diff
@@ -50,12 +60,17 @@ func TestTimer_StopTimer(t *testing.T) {
...
 	time.Sleep(2 * time.Second)
 	tmr.StopTimer()
 	e := tmr.Elapsed()
-	if time.Duration(2*time.Second) > e || e > time.Duration(3*time.Second) {
-		t.Errorf("expected around %s got %s", time.Duration(2*time.Second), e)
+	delayadj := delay
+	if runtime.GOOS == "windows" {
+		delayadj -= windowsInaccuracy
+	}
+	if e < delayadj || e > delay+time.Second {
```
Comment (e-dard) [inline stress/stress_test.go]: It's still not clear to me why we're letting this test pass if the timer is almost a second slower than it should be, regardless of the platform.

Comment (jsternberg) [inline stress/stress_test.go]: Maybe I should make a change to this test to skip it on Windows for now. I'll keep this PR open, but at least we can stop having random test failures.

**Comment by jwilder** [comment]
One of my PRs fixed this in 7aefe1e.  I think this can be closed.

**Pull request closed without merging**